### PR TITLE
pagination

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/BeerOfTheDay/styles.scss [hunk 1]
```diff
@@ -6,28 +6,28 @@
...
-  background-color: white;
+  color:white;
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    outline: 0;
+    font-family: fantasy;
+    border: none;
+    background-color:#3e8e41;
+    font-size: 15px;
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/styles.scss]: Incorrect indentation

File: src/screens/BeersFavourite/styles.scss [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
   font-size: 40px;
-  text-align: center;
-  letter-spacing: 3px;
-  font-family: fantasy;
-  margin-top: 80px;
+    text-align: center;
+    letter-spacing: 3px;
+    font-family: fantasy;
+    margin-top: 80px;
+    color: black;
```
Comment (mi-chal) [inline src/screens/BeersFavourite/styles.scss]: Incorrect indentation

File: src/screens/SearchBeers/styles.scss [hunk 3]
```diff
@@ -61,7 +61,69 @@
 }
 
 .form {
-  width: 500px;
-  height: 150px;
-  margin-bottom: 100px;
+ display: flex;
+ flex-direction: row;
+ margin-left: 230px;
+ margin-top: -25px;
```
Comment (mi-chal) [inline src/screens/SearchBeers/styles.scss]: Incorrect indentation

File: src/screens/WorldBeers/styles.scss [hunk 4]
```diff
@@ -12,24 +13,27 @@
...
-  border-radius: 8px;
-  color: whitesmoke;
+  border: none;
+    padding: 16px 32px;
+    text-decoration: none;
+    cursor: pointer;
+    color: white;
+    font-size: 14px;
+    text-align: center;
+    outline: 0;
```
Comment (mi-chal) [inline src/screens/WorldBeers/styles.scss]: Incorrect indentation

File: src/shared/components/Input/Input.jsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './style.scss';
 import PropTypes from 'prop-types';
 
-const InputComponent = ({ register, className, placeholder, type, name, onChange }) => {
+const InputComponent = ({ value, register, className, placeholder, type, name, onChange }) => {
```
Comment (mi-chal) [inline src/shared/components/Input/Input.jsx]: Do you pass value to `InputComponent` anywhere?

Comment (dwantula) [reply src/shared/components/Input/Input.jsx]: Done

File: src/screens/BeerOfTheDay/styles.scss (hunk 1 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/styles.scss]: Done

File: src/screens/BeersFavourite/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/screens/BeersFavourite/styles.scss]: Done

File: src/screens/SearchBeers/styles.scss (hunk 3 above)
Comment (dwantula) [reply src/screens/SearchBeers/styles.scss]: Done

File: src/screens/WorldBeers/styles.scss (hunk 4 above)
Comment (dwantula) [reply src/screens/WorldBeers/styles.scss]: Done

**Pull request merged**